Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -106,19 +106,33 @@ public async Task<ContestDetailsServiceModel> GetContestDetails(int id)
                 ? competeParticipant
                 : practiceParticipant;
 
-            if (!isLecturerInContestOrAdmin && participantToGetProblemsFrom != null && contestActivityEntity.CanBeCompeted)
+            if (!isLecturerInContestOrAdmin && participantToGetProblemsFrom != null && contestActivityEntity.CanBeCompeted && contestDetailsServiceModel!.IsOnlineExam)
```

Comment:
Everything seems correct for online contests, but for onsite ones the problems are still not visible:
![image](https://github.com/user-attachments/assets/6d8d6fb5-c79b-40c6-856c-bdd28d41f8d1)

user is participant:
![image](https://github.com/user-attachments/assets/3817ca10-2531-4d73-bfa1-435e78079e47)

Reply:
Done.